tasks/page/makeReadme.js

```diff
@@ -60,3 +72,14 @@ function escapeRegExp(string) {
 function addSpacing(string) {
   return string.replace(/[^ `](`[^\`].*?`)/g, ' $1')
 }
+
+function correctImagePaths(string, projectPath) {
+  const imagePaths = /(\()(.+?(\.png|\.jpg|\.gif|\.jpeg|\.bmp))/g
```

Comment:
I think we have some doubling up of this logic [here](https://github.com/kitspace/kitspace/blob/2b56d4d29c87b73a24fa985a922c3bfbea3f30db/tasks/page/makeReadme.js#L47) (also, I think yours is case sensitive). Can you condense these into one?

Reply:
Thinking about it some more, it makes me feel uneasy how this is matched. Could we do it on html img elements post-processing instead? It's also Github specific and we aren't checking for that anywhere?

Reply:
Going to get this merged this week!

I improved the regex matching: https://regex101.com/r/XcdAeP/2
Think this is strong enough?

Not exactly sure how to combine the two you highlighted above. One is checking for the GitHub direct links that include `/blob/master` and the second checking a repo path.

Reply:
Thanks for your effort! I think we should use one regex for finding image paths, and then another to check for `/blob/master`. It makes it easier to add image types later on.

Reply:
https://github.com/kitspace/kitspace/pull/195/files#diff-061cabbd89c63bee0b066d3037b187c2R70-R71

Removed the image file type from the blob url check. Also took out the `/master/` match, as you could also point to a specific branch. I think this would be very rare though.